### PR TITLE
[AttributeBundle] Non translatable attributes

### PR DESCRIPTION
**Pull request by JakobTolkemit**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | #11689
| License         | MIT

Adds a `translatable` flag to attributes. 
Non translatable attributes have *one* value for all locales.

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/Product/Attribute/attributesCollection.html.twig [hunk 1]
```diff
@@ -5,22 +5,34 @@
     {% import _self as self %}
 
     <div class="ui top attached tabular menu">
+
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Product/Attribute/attributesCollection.html.twig:8]:
```suggestion
```

File: src/Sylius/Bundle/CoreBundle/Migrations/Version20171003103916.php [hunk 2]
```diff
@@ -37,7 +37,18 @@ public function up(Schema $schema): void
         $defaultLocale = $this->container->getParameter('locale');
         $productAttributeRepository = $this->container->get('sylius.repository.product_attribute');
 
-        $productAttributes = $productAttributeRepository->findBy(['type' => SelectAttributeType::TYPE]);
+        $productAttributes = $productAttributeRepository
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/Migrations/Version20171003103916.php:40]: The existing migrations shouldn't be changed

Comment (JakobTolkemit) [reply src/Sylius/Bundle/CoreBundle/Migrations/Version20171003103916.php:40]: The problem is, this migration is blocking changes to the attribute mapping as described in: https://github.com/Sylius/Sylius/issues/11592

Comment (JakobTolkemit) [reply src/Sylius/Bundle/CoreBundle/Migrations/Version20171003103916.php:40]: https://github.com/Sylius/Sylius/pull/11624 was actually supposed to have fixed that.

**Comment by GSadee** [comment]
I've pulled your branch and checked these attributes and to be honest, it doesn't look like it's working. When I try to add a new not translatable attribute to the product, It doesn't appear in the list of attributes. Could you provide some behat scenarios for adding a not translatable attribute to the existing product? Similar to [these ones](https://github.com/Sylius/Sylius/blob/master/features/product/managing_products/adding_text_attributes_to_existing_product.feature)

File: src/Sylius/Bundle/CoreBundle/Migrations/Version20171003103916.php (hunk 2 above)
Comment (JakobTolkemit) [reply src/Sylius/Bundle/CoreBundle/Migrations/Version20171003103916.php]: This is now fixed in the master

**Comment by Tomanhez** [comment]
This PR is continue on: https://github.com/Sylius/Sylius/pull/11976

**Pull request closed without merging**